features/git-town/configuration/configuration_automatic.feature

```diff
@@ -27,13 +27,31 @@ Feature: Automatically running the configuration wizard if Git Town is unconfigu
     | git sync-fork      |
 
 
+  Scenario Outline: Proceeding to configuration upon initial config prompt (using prompt's default)
+    When I run `<COMMAND>` and enter "" and "^C"
+    Then I see "Git Town hasn't been configured for this repository."
```

Comment:
I think you should verify that you see the second step in the wizard here, which would prove that the wizard got started and you got past the initial question.